velox/functions/lib/DateTimeUtil.h

```diff
@@ -373,4 +370,30 @@ FOLLY_ALWAYS_INLINE Timestamp addToTimestamp(
   return result;
 }
 
+FOLLY_ALWAYS_INLINE bool isEndDayOfMonth(const std::tm& tm) {
+  return tm.tm_mday == util::getMaxDayOfMonth(getYear(tm), getMonth(tm));
+}
+
+FOLLY_ALWAYS_INLINE double
+monthsBetween(const std::tm& tm1, const std::tm& tm2, bool roundOff) {
```

Comment:
Only SparkSql has this function, we could move it to struct MonthsBetweenFunction